<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FrequencyFileGeneratorDialog</class>
 <widget class="QDialog" name="FrequencyFileGeneratorDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>459</width>
    <height>357</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Frequency File Generator</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_4">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_3">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <layout class="QVBoxLayout" name="verticalLayout">
         <item>
          <widget class="QPushButton" name="inputButton">
           <property name="text">
            <string>Select input</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="outputButton">
           <property name="text">
            <string>Select output</string>
           </property>
          </widget>
         </item>
         <item alignment="Qt::AlignHCenter">
          <widget class="QLabel" name="label">
           <property name="text">
            <string>Morphemizer:</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <item>
          <widget class="QLineEdit" name="inputFileLiineEdit"/>
         </item>
         <item>
          <widget class="QLineEdit" name="outputFileLineEdit"/>
         </item>
         <item alignment="Qt::AlignLeft">
          <widget class="QComboBox" name="comboBox">
           <property name="minimumSize">
            <size>
             <width>150</width>
             <height>0</height>
            </size>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <property name="topMargin">
        <number>10</number>
       </property>
       <item>
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>Minimum occurrence:</string>
         </property>
        </widget>
       </item>
       <item alignment="Qt::AlignLeft">
        <widget class="QSpinBox" name="minOccurrenceSpinBox">
         <property name="minimum">
          <number>1</number>
         </property>
         <property name="maximum">
          <number>1000</number>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_5">
       <property name="topMargin">
        <number>10</number>
       </property>
       <item>
        <widget class="QCheckBox" name="squareBracketsCheckBox">
         <property name="text">
          <string>Ignore content in square brackets []</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="roundBracketsCheckBox">
         <property name="text">
          <string>ignore content in round brackets（）</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="slimRoundBracketsCheckBox">
         <property name="text">
          <string>ignore content in slim round brackets ()</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="namesMorphemizerCheckBox">
         <property name="text">
          <string>Ignore names found by morphemizer</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="namesFileCheckBox">
         <property name="text">
          <string>Ignore names found in &quot;names.txt&quot;</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="numbersCheckBox">
         <property name="text">
          <string>Ignore numbers</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <spacer name="verticalSpacer">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_7"/>
     </item>
     <item alignment="Qt::AlignHCenter">
      <widget class="QPushButton" name="createFrequencyFileButton">
       <property name="styleSheet">
        <string notr="true">margin-top: 10px;
margin-bottom: 10px;
padding: 5px;</string>
       </property>
       <property name="text">
        <string>Create Frequency File</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
